backend/tests/db/test_worker.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from sqlalchemy.orm import Session as OrmSession
+
+from mirrors_qa_backend.db import models, worker
+
+
+def test_create_worker(dbsession: OrmSession, tmp_path: Path):
+    worker_id = "test"
+    countries = [
+        models.Country(code="ng", name="Nigeria"),
+        models.Country(code="fr", name="France"),
+    ]
+    dbsession.add_all(countries)
+
+    private_key_filename = tmp_path / "key.pem"
```

Comment:
```suggestion
    private_key_fpath = tmp_path / "key.pem"
```

It's not a filename but a full path and `fpath` is our convention in this case